src/RageUtil.cpp

```diff
@@ -1819,8 +1819,13 @@ void UnicodeUpperLower( wchar_t *p, std::size_t iLen, const unsigned char pMappi
 	wchar_t *pEnd = p + iLen;
 	while( p != pEnd )
 	{
-		if( *p >= 0 && *p < 256 )
+		#if WCHAR_MIN == 0

```

Comment:
Let's be consistent with the condition for this macro with the other one above. Pick whichever one you prefer (I might prefer the != 0 version). Add a comment as well as well (because for people who don't know about WCHAR_MIN, it's not obvious to what these statements are doing.

Reply:
I'm afraid I don't understand why wouldn't this construction emit a warning: https://godbolt.org/z/x1ToWvE46

Reply:
Oh right I was misunderstanding the warning. it's harmless here though, compared to the bone index that would always be invalid and crash.

(Just dropping the preprocessor directive would be good then, or just a readable / consistent version since it's an odd thing to see at first sight)

Reply:
I added a comment and used the same preprocessor directives in both affected places. Let me know if you come up with something more readable